cmd/lk/cloud.go

```diff
@@ -322,16 +322,34 @@ func tryAuthIfNeeded(ctx context.Context, cmd *cli.Command) error {
 		return err
 	}
 
+	// make sure name is unique
+	var name = ak.ProjectName
+	if cliConfig.ProjectExists(name) {
+		if err := huh.NewInput().
```

Comment:
for some reasons.. every time I see this package, I can't help but to think of @cacheonly and Noah's joke

Reply:
SAME https://github.com/livekit/livekit-cli/pull/365#issue-2401952169

Reply:
What was Noah's joke though?